.github/workflows/build-and-release-nhost-run.yaml

```diff
@@ -16,6 +16,15 @@ on:
         type: string
         required: false
         default: .
+      DOCKER_FILE:
+        description: Path to the Dockerfile
+        type: string
+        required: false
```

Comment:
should this have a default?

Reply:
I am mostly wondering if specifying it as empty will be treated as `""` and cause some issues. The consuming action says the `file` isn't required but not being required and being an empty string might be different things.

Reply:
From what I understand this is acceptable. But not sure 100%.  I can make it `./Dockerfile` by default

Reply:
Problem that by default it says {{context}}/Dockerfile, but I'm not sure that it's acceptable to use inputs in the inputs definition :P

Reply:
yeah, let's make it "Dockerfile". That will probably translate to context/Dockerfile as the filepath should be relative to the context

Reply:
added @dbarrosop